### PR TITLE
Upload to pypi daily

### PR DESCRIPTION
**Pull request by zhanghang1989**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/pythonpublish.yml [hunk 1]
```diff
@@ -7,9 +7,8 @@
 name: Upload Python Package
 
 on:
-  push:
-    branches: 
-      - master
+  schedule:
+    - cron: "0 12 * * *"
```
Comment (zhreshold) [inline .github/workflows/pythonpublish.yml:11]: This is the UTC time, I personally prefer 59 8 * * *  which starts 23:59 in PST

Comment (zhanghang1989) [reply .github/workflows/pythonpublish.yml]: Good suggestion

**Comment by szha** [comment]
Job PR-413-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-413/1/index.html

**Comment by szha** [comment]
Job PR-413-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-413/2/index.html

**Pull request merged**